### ISSUE TITLE
Coloring started tasks

### ISSUE THREAD

**Issue by fabiospampinato** [fabiospampinato/vscode-todo-plus#292]
As implemented in #268 this extension could color started tasks with a different color.

Preview: https://pasteboard.co/JqO134S.gif

Is that something that you'd want? If so add a "👍" to this post.

**Comment by CaptnCodr** [comment]
An extra symbol would be nice for the "started" state.
Maybe "⭕" as default. "Circle" because it's in progress.

**Comment by fsevenm** [comment]
> An extra symbol would be nice for the "started" state.
> Maybe "o" as default. "Circle" because it's in progress.

Nice idea, IMO

**Comment by CaptnCodr** [comment]
> Maybe "⭕" as default. "Circle" because it's in progress.

Additionaly, `[o]` and `[O]` could also be default in this category.

**Comment by fsevenm** [comment]
I've updated the fork, updated the icon of started tasks and made it counted in statistic. These are some images that may help me visualize it.
![Screenshot-20201230094510-898x174](https://user-images.githubusercontent.com/29963765/103326510-9ecbd480-4a8b-11eb-8fd2-6787e4431120.png)
The classic version:
![Screenshot-20201230094223-827x166](https://user-images.githubusercontent.com/29963765/103326528-b0ad7780-4a8b-11eb-8333-05dd2aaa2188.png)

You can try this enhancement now btw, it's at https://marketplace.visualstudio.com/items?itemName=fsevenm.vscode-todo-plus. Yes, I publish it in marketplace, but will take it down asap when the pr gets merged.

**Comment by CaptnCodr** [comment]
@fsevenm Could you also add the `todo.timekeeping.started.enabled` property to configure, as like the `created` and `finished` property? 
To have the option to enable/disable the `@started` tag on the task and consistency over the states. I don't always have activated the time and don't want to have any tags on them. That would be great!

Overall it works fine on my machine. Thanks for the practical "preview".

**Comment by fsevenm** [comment]
Yeah, forgot that one. It's nice idea. I'll update the extension soon.

**Comment by CaptnCodr** [comment]
In addition: there's something different than the other states, see:
![image](https://user-images.githubusercontent.com/4574031/103359681-950c9600-4ab8-11eb-8abf-8e872b547ac6.png)
The dots configured color before the task when marked.

**Comment by fsevenm** [comment]
@CaptnCodr Sorry, just come back here after a couple of weeks. I've just pushed an update to the extension. It allows you to disable `@started` tag via config. 

And about the dots problem, I cannot reproduce that. It's just fine on mine. See this,
![image](https://user-images.githubusercontent.com/29963765/104392417-fafb2200-557c-11eb-8f60-a9ce5d068599.png)

**Comment by CaptnCodr** [comment]
This is great!

When you select the whole line then the colored dots appear:
![image](https://user-images.githubusercontent.com/4574031/104420141-b7eb7f80-5579-11eb-9522-7f32b81984c8.png)

**Comment by fsevenm** [comment]
@CaptnCodr It's weird, I enabled "Render Whitespace" and it's just fine.
![Peek 2021-01-14 17-43](https://user-images.githubusercontent.com/29963765/104573646-1571f000-5690-11eb-9b24-cd4ae932b754.gif)

I am using VS Code 1.52.1 on linux machine. What's yours?

**Comment by CaptnCodr** [comment]
Your newest preview (4.19.3) works as expected. Approved from me. 🙂 

Thank you, @fsevenm. 👍

**Comment by fsevenm** [comment]
Thank You @CaptnCodr. Glad to hear it works now. I was on Windows in virtualbox, so I was able to reproduce your issue and made a fix for that. It's only on Windows I guess.

**Comment by House-of-Leaves** [comment]
Thank you for this; having the started tasks coloured with a different icon is really helpful!

The only thing I've noted is that the number on the line for a Project that shows the number of pending tasks only takes into account tasks not started now, started tasks are not included in the count. I believe originally pending and started tasks were included in the count?

Apologies if I have missed this and it is switchable somewhere. Many thanks.

**Comment by fsevenm** [comment]
@House-of-Leaves  Did you mean number of pending tasks in status bar?

**Comment by House-of-Leaves** [comment]
@fsevenm Thank you for the swift reply. Sorry, I should have explained better.

Previously a project listing would have looked something like this:
![todo-before](https://user-images.githubusercontent.com/79336499/108606929-1e539100-73b5-11eb-94d5-bb58525af03c.png)

The top line with the project title would include the number of outstanding tasks in parentheses, in this case **(2)**

When using the new version with the different icon for started tasks, the same todo now looks like this:
![todo-after](https://user-images.githubusercontent.com/79336499/108607027-c10c0f80-73b5-11eb-986d-1b0485519fc5.png)

The number of outstanding tasks now shows **(1)**

Many thanks.

**Comment by House-of-Leaves** [comment]
I've figured out I can work around this by changing the settings to something like:

`"todo.statistics.project.text": "(P:[pending] S:[doing])`

It is whether it's possible to have a statistics token that is the sum of [pending] and [doing], as I presume [finished] is the sum of [done] and [cancelled]
